apps/web/src/pages/Profile/AccountSettings/NotificationSettings.tsx

```diff
@@ -0,0 +1,310 @@
+  const [, executeMutation] = useMutation(UpdateIgnoredNotifications_Mutation);
+
+  const methods = useForm<FormValues>({
+    defaultValues: dataValuesToFormValues({
+      ignoredEmailNotifications,
+      ignoredInAppNotifications,
+    }),
+  });
+
+  const { handleSubmit } = methods;
+
+  const handleError = () => {
+    toast.error(
+      intl.formatMessage({
+        defaultMessage: "Error: could not update settings",
+        id: "EG5WpQ",
+        description:
+          "Message displayed when an error occurs while updating notification settings.",
+      }),
+    );
+  };
+
+  const handleFormSubmit: SubmitHandler<FormValues> = async (
+    values: FormValues,
+  ) => {
+    const data = formValuesToData(values);
+    await executeMutation(data)
+      .then((result) => {
+        if (result.data?.updateIgnoredNotifications) {
+          toast.success(
+            intl.formatMessage({
+              defaultMessage: "Successfully updated settings",
+              id: "ennrdC",
+              description:
+                "Message displayed when applicant updated their notification settings.",
+            }),
+          );
+        } else {
+          handleError();
+        }
+      })
+      .catch(() => {
+        handleError();
+      });
+  };
+
+  const formFields = [
+    {
+      id: "systemMessages",
+      name: "systemMessages",
+      legend: intl.formatMessage({
+        defaultMessage: "System messages",
+        id: "HBmwNb",
+        description: "Title for system messages checklist",
+      }),
+      subtitle: intl.formatMessage({
+        defaultMessage:
+          "These notifications are required and include email verification and other important communications.",
+        id: "blvpQr",
+        description: "Subtitle for system messages checklist",
+      }),
+      disabled: true,
+    },
+    {
+      id: "applicationUpdates",
+      name: "applicationUpdates",
+      legend: intl.formatMessage({
+        defaultMessage: "Application updates",
+        id: "v21LPW",
+        description: "Title for application updates checklist",
+      }),
+      subtitle: intl.formatMessage({
+        defaultMessage:
+          "Get notified about upcoming application deadlines, status changes, and more.",
+        id: "XzyFuo",
+        description: "Subtitle for application updates checklist",
+      }),
+    },
+    {
+      id: "jobAlerts",
+      name: "jobAlerts",
+      legend: intl.formatMessage({
+        defaultMessage: "Job alerts",
+        id: "fRoNIK",
+        description: "Title for job alerts checklist",
+      }),
+      subtitle: intl.formatMessage({
+        defaultMessage:
+          "Get notified when a new job is posted to the platform.",
+        id: "u3CPra",
+        description: "Subtitle for job alerts checklist",
+      }),
+    },
+  ];
+
+  return (
+    <FormProvider {...methods}>
+      <form onSubmit={handleSubmit(handleFormSubmit)}>
+        <Card title="" color="white">
```

Comment:
I would just use `CardBasic` here instead so you don't need to pass in an empty title prop.

Reply:
Fixed by [bd66609](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/bd66609eb541fa39fcdcaabdadde6ffae463366c)